packages/@aws-cdk/aws-apigateway/lib/model.ts

```diff
@@ -1,46 +1,141 @@
-export interface IModel {
-    readonly modelId: string;
-}
+import { Construct, PhysicalName, Resource } from '@aws-cdk/cdk';
+import { CfnModel, CfnModelProps } from './apigateway.generated';
+import jsonSchema = require('./json-schema');
+import { IRestApi, RestApi } from './restapi';
 
-/**
- * Represents a reference to a REST API's Empty model, which is available
- * as part of the model collection by default. This can be used for mapping
- * JSON responses from an integration to what is returned to a client,
- * where strong typing is not required. In the absence of any defined
- * model, the Empty model will be used to return the response payload
- * unmapped.
- *
- * Definition
- * {
- *   "$schema" : "http://json-schema.org/draft-04/schema#",
- *   "title" : "Empty Schema",
- *   "type" : "object"
- * }
- *
- * @see https://docs.amazonaws.cn/en_us/apigateway/latest/developerguide/models-mappings.html#models-mappings-models
- */
-export class EmptyModel implements IModel {
-    public readonly modelId = 'Empty';
+export interface IModelRef {
```

Comment:
Please leave this `IModel`. This is out convention for interface type names

Reply:
Changed